.github/workflows/build.yml

```diff
@@ -120,6 +123,20 @@ jobs:
           username: ${{ secrets.RELEASE_DOCKERHUB_ACCOUNT }}
           password: ${{ secrets.RELEASE_DOCKERHUB_TOKEN }}
 
+      - name: Arch Runner is Matrix
+        id: arch_runner_equals_matrix
+        run: |
+          RUNNER_ARCH=${{ runner.arch }}
+          [ "$RUNNER_ARCH" = "X64" ] && RUNNER_ARCH="amd64"
+          [ "$RUNNER_ARCH" = "ARM64" ] && RUNNER_ARCH="arm64"
+          [ "$RUNNER_ARCH" = "X86" ] && RUNNER_ARCH="i386"
+          [ "$RUNNER_ARCH" = "ARM" ] && RUNNER_ARCH="arm"
+          MATCHED="false"
```

Comment:
But where do we set it to `true` if so?

Reply:
Doh! I forgot to save and commit and push my last change!

Reply:
Nice catch @OhmSpectator , thank you.